package.json

```diff
@@ -115,6 +115,7 @@
     "node-sass-chokidar": "1.3.0",
     "npm-snapshot": "1.0.3",
     "prettier": "1.13.4",
+    "pretty-quick": "^1.8.0",
```

Comment:
We're using fixed versions everywhere, so unless we're changing our policy on versions I guess we should have "1.8.0" here.